src/core/AnimatedCallFunc.js

```diff
@@ -39,15 +40,19 @@ class AnimatedCallFunc extends AnimatedNode {
   }
 
   beginContext() {
+    this._previousCallID = getCallID();
+    setCallID(getCallID() + '/' + this.__nodeID);
+
     this._params.forEach((param, index) => {
-      param.beginContext(this._args[index]);
+      param.beginContext(this._args[index], this._previousCallID);
     });
   }
 
   endContext() {
     this._params.forEach((param, index) => {
       param.endContext();
     });
+    setCallID(this._previousCallID);
```

Comment:
just wondering if `setCallID(getCallID.split('/').slice(0, -1).join('/'))` is not enough? I know I used it in native code, but maybe it's worth refactoring

Reply:
I think I made it like these because keys of dictionaries in ObjC are objects not actually texts and then I followed up with Java impl in the same way.

Reply:
I'm attempting to make the JS logic match the native logic exactly. Makes it easier to spot bugs. Would probably be worth updating it everywhere in another PR.

Reply:
sure